app/models/anime.rb

```diff
@@ -260,105 +335,103 @@ def self.create_or_update_from_hash(hash)
       character = Character.create_or_update_from_hash ch
       if ch[:voice_actors].length > 0
         ch[:voice_actors].each do |actor|
-          Casting.create_or_update_from_hash actor.merge({
+          Casting.create_or_update_from_hash actor.merge(
             featured: ch[:featured],
             character: character,
             castable: anime
-          })
+          )
         end
       else
-        Casting.create_or_update_from_hash({
+        Casting.create_or_update_from_hash(
           featured: ch[:featured],
           character: character,
           castable: anime
-        })
+        )
       end
     end
     anime
   end
 
   def show_type_enum
-    ["OVA", "ONA", "Movie", "TV", "Special", "Music"]
+    self::SHOW_TYPES
   end
 
   def status
-    # If the started_airing_date is in the future or not specified, the show hasn't
-    # aired yet.
-    if started_airing_date.nil? or started_airing_date > Time.now.to_date
-      return "Not Yet Aired"
+    # If the started_airing_date is in the future or not specified,
+    # the show hasn't aired yet.
+    if started_airing_date.nil? || started_airing_date > Time.zone.now.to_date
```

Comment:
I don't think we actually need to use `Time.zone` since we work *only* in UTC.  The only part which deals with timezones is the Ember app